packages/eslint-plugin/docs/rules/space-before-blocks.md

```diff
@@ -0,0 +1,58 @@
+# Enforces consistent spacing before blocks (`space-before-blocks`)
```

Comment:
```suggestion
# space-before-blocks

Enforces consistent spacing before blocks.
```

https://github.com/typescript-eslint/typescript-eslint/runs/5175689231?check_suite_focus=true

We updated this format recently in #4367.